### PR TITLE
Fix for non-running MPI jobs under qrsh

### PR DESCRIPTION
**Pull request by jkwmoore**
Minor edits to ensure interactive sessions have working MPI.

### CODE REVIEW

File: sharc/software/parallel/mvapich2-intel.rst [hunk 1]
```diff
@@ -73,7 +73,8 @@ To build an MPI program (e.g. :download:`mpi_hello_world.c`)::
     ssh user@sharc  
 
     # Start an interactive session 
-    qrsh 
+    # Note you must use qrshx not qrshx
```
Comment (willfurnass) [inline sharc/software/parallel/mvapich2-intel.rst:76]: There's a typo in this line; I'd just omit the whole line.

File: sharc/software/parallel/openmpi-gcc.rst [hunk 2]
```diff
@@ -43,8 +43,9 @@ In more detail ::
     # Connect to ShARC
     ssh user@sharc  
 
-    # Start an interactive session from which we can run MPI processes using a core on each of four nodes
-    qrsh -pe mpi 4
+    # Start an interactive session from which we can run MPI processes using a core on each of four nodes 
+    # Note you must use qrshx not qrshx
```
Comment (willfurnass) [inline sharc/software/parallel/openmpi-gcc.rst:47]: Again, I'd omit this line.

**Review by willfurnass — CHANGES_REQUESTED** [review]

File: sharc/software/parallel/mvapich2-intel.rst (hunk 1 above)
Comment (jkwmoore) [reply sharc/software/parallel/mvapich2-intel.rst:76]: Fair enough - I shall pull the line entirely. Typical that muscle memory keeps having me add the x even when I don't want it.

File: sharc/software/parallel/openmpi-gcc.rst (hunk 2 above)
Comment (jkwmoore) [reply sharc/software/parallel/openmpi-gcc.rst:47]: Same as above.

**Review by willfurnass — APPROVED** [review]

**Pull request merged**